pallets/runtime/src/benches/asset.rs

```diff
@@ -0,0 +1,243 @@
+use crate::asset::*;
+use codec::Encode;
+use frame_benchmarking::{account, benchmarks};
+use frame_support::{traits::Currency, StorageValue};
+use frame_system::RawOrigin;
+use polymesh_primitives::{AccountKey, AuthorizationData, IdentityId, Signatory, Ticker};
+use polymesh_runtime_balances as balances;
+use polymesh_runtime_identity as identity;
+use sp_std::{convert::TryFrom, iter, prelude::*};
+
+const SEED: u32 = 0;
+const MAX_USER_INDEX: u32 = 1_000;
+const MAX_TICKER_LENGTH: u8 = 12;
+const MAX_NAME_LENGTH: u32 = 64;
+
+fn make_account<T: Trait>(
```

Comment:
duplicated code from `benchmarking.rs`

Reply:
It's slightly different but you are right, both `identity` and `asset` benchmarks can use utility functions from a library.